### DISCUSSION TITLE
Why is main content getting wrapped inside annoucementBar class?

### DISCUSSION THREAD

**Discussion by kdb13** [facebook/docusaurus#9206] [Q&A]
I am displaying an announcement bar using the `docusaurus.config.js` file and am deploying the site to AWS. When the site is deployed, the content is visible fine when I visit the home or the index page. But if I try to visit an inner doc by its absolute URL from the browser the content looks out of order and the main content is wrapped inside `announcementBar*` tag when I inspected the rendered page. And, that's what is causing the problem.

Here is the rendered HTML in the browser:
![DevTools screenshot](https://github.com/facebook/docusaurus/assets/36751623/85da0c40-46f8-43c1-b1c5-076e3625ac38)

Now, the thing is that this problem only arises once my site live on AWS. When, I run it locally with `start` and `build`, there is no such problem and the annoucement bar displays correctly.

**Comment by slorber** [comment]
I can't tell anything without access to your site url

Are you using Docusaurus v2.4.1?
